guides/doc-Upgrading_and_Updating/topics/upgrading_capsule_server.adoc

```diff
@@ -116,7 +116,8 @@ Ensure {SmartProxy} has access to `{RepoRHEL7ServerSatelliteMaintenanceProductVe
 +
```

Comment:
The line above this looks redundant now: that tells the user to ensure access and then run a command to ensure access.

Reply:
Addressed :+1: